lib/sidekiq/processor.rb

```diff
@@ -234,5 +234,14 @@ def cloned(thing)
       Marshal.load(Marshal.dump(thing))
     end
 
+    def constantize(str)
+      names = str.split('::')
+      names.shift if names.empty? || names.first.empty?
```

Comment:
But the first condition is doing nothing in this case, just the second one. Actually, for this specific case of an empty string before split, not even the second one, but I understand the second one is important for `'Class'.split("::")`.